build.gradle

```diff
@@ -212,6 +212,17 @@ subprojects {
 
 
   ext {
+    applyIntellijPlugin = { Map params ->
+      apply plugin: 'org.jetbrains.intellij'
+      disableIntellijTasks()
```

Comment:
If `disableIntellijTasks` are only called here (which I think is now the case), then consider inlining